Package.swift

```diff
@@ -11,4 +12,24 @@
 
 import PackageDescription
 
-let package = Package(name: "DeviceKit")
+let package = Package(
+    name: "DeviceKit",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "DeviceKit",
+            targets: ["DeviceKit"])
```

Comment:
Could you add the platforms DeviceKit supports here?
https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md#package

Reply:
```suggestion
            targets: ["DeviceKit"]),
            platforms: [
                .iOS(.v8_0),
                .tvOS(.v9_0),
                .watchOS(.v2_0)
            ]
```